aten/src/ATen/native/Distributions.cpp

```diff
@@ -188,8 +188,8 @@ Tensor _standard_gamma_grad_cpu(const Tensor& self, const Tensor& output) {
   Tensor ret = self.type().tensor(self.sizes());
   AT_DISPATCH_FLOATING_TYPES(self.type(), "_standard_gamma_grad", [&] {
     CPU_tensor_apply3<scalar_t, scalar_t, scalar_t>(ret, self, output,
-      [](scalar_t& ret_val, const scalar_t& self_val, const scalar_t &output_val) {
-         ret_val = standard_gamma_grad_one(self_val, output_val);
+      [](scalar_t* ret_val, const scalar_t* self_val, const scalar_t* output_val) {
+         ret_val[0] = standard_gamma_grad_one(self_val[0], output_val[0]);
```

Comment:
Why are we changing references to pointers? The intent was clearer previously - you want to modify a *single* element.

Reply:
Yes, because the functions will soon be required to take a size

Reply:
Hmm that's not great. I think we should still keep a syntax (perhaps an alternative version of the apply functions) that lets us avoid having an explicit `for` loop, because we won't be vectorizing all of our ops.

Reply:
It's also possible (and imo necessary) to support both.